rnaseq-cgl-pipeline/test.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2.7
+# John Vivian
+import subprocess
+import unittest
+
+
+class TestRNASeqPipeline(unittest.TestCase):
+
+    def test_docker_call(self):
+        out, err = check_docker_output(tool='quay.io/ucsc_cgl/rnaseq-cgl-pipeline')
```

Comment:
I would test the `-v` checks in wrapper.py here as well. Go as deep as you can without spending too much time coding or actually running the pipeline.